### PR TITLE
V3: Adds ui for legend point color setting in inspector palette

### PR DESCRIPTION
**Pull request by eireland**
Adds color inputs for categorical and numeric legends in the brush palette.
Brush and ruler inspector palette open and close when brush or ruler inspector panel buttons are clicked.
Inspector palettes close once inspector panel is closed, and stays closed when inspector panel is re-opened.

### CODE REVIEW

**Review by bfinzer — APPROVED** [review]
👍🏻LGTM

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Code looks fine -- my main suggestion is to add some comments to some parts that weren't immediately obvious to me, e.g.
- `-webkit`-only CSS
- add `TODO` comments where the code will have to change once the necessary model support for legend colors is in place. I was briefly confused as to how calling `setPointColor()` could possibly change a legend category color before realizing that it's not supposed to (yet).

File: v3/src/components/graph/components/inspector-panel/point-format-panel.scss [hunk 1]
```diff
@@ -7,4 +7,26 @@
...
+    border-radius: 0.175rem;
+    background-color: lightgray;
+  }
+  &::-webkit-scrollbar {
+    width: 0.35rem;
+    border-radius: 0.125rem;
+  }
+  &::-webkit-scrollbar-thumb {
+    border-radius: 0.175rem;
+    background-color: gray;
```
[the remark this answers is not in the document]
Comment (kswenson) [reply v3/src/components/graph/components/inspector-panel/point-format-panel.scss:40]: I don't think my questions are answered by the new comment.

**Review by kswenson — APPROVED** [review]
👍 I think it would still be good to improve the comment for the `-webkit`-only CSS rules, but I'll approve and leave that to your discretion.

**Pull request merged**